### PR TITLE
CU remote install: Support for spaces in 'launchPath' and 'patchPath'.

### PR DESCRIPTION
**Pull request by gszdev**
Use single quotes to support spaces in 'launchPath' and 'patchPath'.

### CODE REVIEW

File: AutoSPUpdaterModule.psm1 [hunk 1]
```diff
@@ -137,14 +137,18 @@ function Install-Remote
             {
                 $versionSwitch = "-Version 2"
             }
-            else {$versionSwitch = ""}
```
Comment (brianlala) [inline AutoSPUpdaterModule.psm1:140]: Can we leave this the way it was just for readability/formatting purposes?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```
Comment (brianlala) [inline .gitignore:1]: Not sure this is needed in the project at this time, for simplicity can we leave it out?

File: AutoSPUpdaterModule.psm1 (hunk 1 above)
Comment (gszdev) [reply AutoSPUpdaterModule.psm1:140]: Sure, leave it.

File: .gitignore (hunk 2 above)
Comment (gszdev) [reply .gitignore:1]: Without the single quotes spaces in the path variables are interpreted as parameter separator. This causes errors. 
You are the owner of the GitHub repository, so you can choose. ;-)
If you leave it out, I'll keep my version. ;-)

**Comment by gszdev** [comment]
Done by commit: 1fea58605a04e86590525a6cd9bf1de2f78cad9f

Syntax/format and other fixes
Replace Get-WmiObject with Get-CimInstance
Better handling of paths [AutoSPUpdaterModule.psm1#L146](https://github.com/brianlala/AutoSPUpdater/blob/3f6cd4d005647934ffa019738c7d353d68059f1c/AutoSPUpdaterModule.psm1#L146)

**Pull request closed without merging**